docs/es/features/oauth.md

```diff
@@ -1,20 +1,27 @@
 # OAuth
 
-Puedes configurar la autenticación con servicios externos usando OAuth, por ahora están soportados Twitter, Facebook y Google.
+Puedes configurar la autenticación con servicios externos usando OAuth, actualmente soportan Twitter, Facebook, Google y Wordpress.
```

Comment:
"actualmente soportan" sounds strange :thinking:. I'm not sure who the subject of that sentence is.